appveyor.yml

```diff
@@ -59,7 +59,7 @@ for:
   - sh: cp ../resources/pictures/carton.png milkytracker.png
   - sh: cp ../resources/milkytracker.desktop .
   - sh: sed -i 's|MilkyTracker|Milkytracker|g;s|Comment=.*|Comment=A Portable SampleTracker DAW|g' milkytracker.desktop
-  #- sh: cp ../resources/milkytracker.appdata /usr/share/metainfo
+  #- sh: cp ../resources/org.milkytracker.MilkyTracker.metainfo.xml /usr/share/metainfo
```

Comment:
they should be installed when running install. (ie if they are not then it should be fixed to do so)

Reply:
I am not the MilkyTracker contributor and don't want to modify their own CI/AppImage build manifests. Just fixing the commented out line should be enough for now, if MilkyTracker developers want, they can fix the rest themselves.